### PR TITLE
First Capstone Project

### PR DESCRIPTION
**Pull request by basitali111**
-Making first capstone by using following things
-Adding Html for structure 
-Adding javascripts for dynamics
-Adding css for error
-adding linters for error checking

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,84 @@
...
+## Built With
+​
+- HTML
+- CSS
+- JavaScript
+​
+​
+## Live Demo
+​
+[Live Demo Link]()
```
Comment (sja-thedude) [inline README.md]: - [ ] Kindly make sure to deploy your capstone with Github Pages and provide the link here.
- [ ] The project is deployed properly, all pages, links, and images work as expected.

File: about.html [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+                <div class="logout">
+                  <a href="#">  <i class="fa fa-facebook-f"  id="facebook"></i></a>
+                  <a href="#"></a> <i class="fa fa-twitter" ></i></a>
+                  <a href="#" class="list">Logout</a>
+                  <a href="#" class="list">Mypage</a>
+                  <a href="#" class="list">Korean</a>
+                </div>
+            </div>
+               <div class="navbar-container">
+                  <div class="logo"><img src="Capstone/Header/logo.png" alt="simmitlogo"></div>
```
Comment (sja-thedude) [inline about.html]: - [ ] Kindly make sure to add the link to the home page and display it when the logo is clicked.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,936 @@
...
+  max-width: 100%;
+  overflow-x: hidden;
+}
+
+/* .logout{
+    display: none;
+}
+.nav-list{
+    display: none;
+} */
```
Comment (sja-thedude) [inline style.css]: - [ ] Kindly remove the unused code for clean code practices.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+              <div class="text-speaker">
+                <h4>Yochai bankler</h4>
+                 <p class="speaker-para">Lorem ipsum dolor sit amet consectetur adipisicing elit. Expedita.</p>
+                 <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Iusto ab porro quibusdam.</p>
+              </div>
+             </div>
+             <div class="speaker-close">
+              <div class="box-image">
+               <img src="Capstone/speakers/speaker1.png" alt="speaker1" class="speaker-image">
+              </div>
```
Comment (sja-thedude) [inline index.html]: - [ ] Kindly remove unused code for the clean code purpose.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+                 <img src="Capstone/speakers/speaker1.png" alt="speaker1" class="speaker-image">
+                </div>
+               <div class="text-speaker">
+                  <h4>yochai bankler</h4>
+                  <p class="speaker-para">Lorem ipsum dolor sit amet consectetur adipisicing elit. Expedita.</p>
+                  <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Iusto ab porro quibusdam.</p>
+               </div>
+             </div>
+             <div class="more-container">
+               <a href="#" class="more-btn">More <img src="capstone/Header/arrow_down.png" alt=""></a>
```
Comment (sja-thedude) [inline index.html]: - [ ] Kindly remove this section. You have to add it via javascript, same as how you have added it to the desktop version.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,183 @@
...
+                <div class="logo"><img src="Capstone/Header/logo.png" alt="simmitlogo"></div>
+                <div ><img src="capstone/Header/close_mobile_menu.png" alt="close-icon" id="close-icon"></div>
+                  <div class="nav-list">
+                    <a href="about.html" class="nav-menu">About</a>
+                    <a href="#"class="nav-menu">Program</a>
+                    <a href="#"class="nav-menu">Join</a>
+                    <a href="#"class="nav-menu">Sponsor</a>
+                    <a href="#"class="nav-menu">News</a>
+                    <a href="index.html"class="nav-menu" id="cc">CC Campaign</a>
+                 </div>
```
Comment (sja-thedude) [inline index.html]: - [ ] Make sure you have a home page link from the mobile version, Currently, the CC campaign works as coming back to the home page. You can add an option for the mobile version popup menu.

File: main.js [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+        <p class="speaker-para">Lorem ipsum dolor sit amet consectetur adipisicing elit. Expedita.</p>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Iusto ab porro quibusdam.</p>
+     </div>
+   </div>
+   <div class="more-container">
+     <a href="#" class="more-btn">More <img src="capstone/Header/arrow_down.png" alt=""></a>
+
+   </div>
+  </div>
+  `;
```
Comment (sja-thedude) [inline main.js:72]: - [ ] Kindly make sure to implement more button functionality for the mobile version as suggested in the attached loom video.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,183 @@
...
+                 <h3 class="heading-box">CC Exhibition</h3>
+                 <p class="box-para">Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+               </div>
+               <div class="grid-box">
+                 <img src="Capstone/Icons/program_icon_03.png" alt="lecture" class="image-box">
+                 <h3 class="heading-box">Forum</h3>
+                 <p class="box-para">Lorem ipsum dolor sit amet consectetur adipisicing elit. </p>
+               </div>
+               <div class="grid-box">
+                 <img src="Capstone/Icons/program_icon_04.png" alt="lecture" class="image-box">
```
Comment (sja-thedude) [inline index.html:74]: - [ ] Kindly follow the same height for the mobile version as shown in the attached loom video.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @basitali111, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ All linters passing
✔️ Great job so far
✔️ Correct Gitflow

## Required 
- [ ] Kindly provide the link for the loom video in the pull request or in the readme. You have just added the link in microverse dashboard.
- Video presentation:
- [ ] Kindly make sure that your loom video contains a demo of the project, showing the pages in different screen sizes.
- [ ] Kindly watch the attached loom video [here](https://www.loom.com/share/5f2d5f68ab0b44789bfb6fb96d1d0f16) to make all the changes.
- [ ] Kindly follow the spacing, and colour as per shown in the loom video.
- [ ] Kindly implement all the changes suggested in the loom video attached.
- [ ] _Check the comments under the review and in the loom video._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Deepakdanger — APPROVED** [review]
Hi @basitali111, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**